<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fDatiGrafoCasuale</class>
 <widget class="QDialog" name="fDatiGrafoCasuale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Random graph</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>images/molecule.png</normaloff>images/molecule.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Vertices</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="eNodiDa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>9999</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="eNodiA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2</number>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Edges</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="eArchiDa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>49999</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="eArchiA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>50000</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>191</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Edges per vertex</string>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>From</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>20</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>to</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="eArchiPerNodoDa">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>199</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="eArchiPerNodoA">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>200</number>
    </property>
    <property name="value">
     <number>15</number>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>fDatiGrafoCasuale</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>fDatiGrafoCasuale</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eArchiA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fDatiGrafoCasuale</receiver>
   <slot>ArchiCambiati()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>147</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eArchiDa</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fDatiGrafoCasuale</receiver>
   <slot>ArchiCambiati()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>209</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eNodiDa</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fDatiGrafoCasuale</receiver>
   <slot>ArchiCambiati()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>44</y>
    </hint>
    <hint type="destinationlabel">
     <x>211</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eNodiA</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fDatiGrafoCasuale</receiver>
   <slot>ArchiCambiati()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>eArchiPerNodoDa</sender>
   <signal>valueChanged(int)</signal>
   <receiver>fDatiGrafoCasuale</receiver>
   <slot>ArchiCambiati()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>57</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>210</x>
     <y>173</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ArchiCambiati()</slot>
 </slots>
</ui>
